cmd/root.go

```diff
@@ -19,6 +19,8 @@ var (
 	BinaryName string
 	// Out is used to write to the required writer.
 	Out io.Writer
+	// In used for testing
```

Comment:
This should be a full sentence, with punctuation.

```
// In is used to provide fake input during testing.
```
(or something like that).